src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+  const price = 40 * days;
+  const longVacation = 7;
+  const shotVacation = 3;
```

Comment:
The variable name `shotVacation` seems to be a typo. Consider renaming it to `shortVacation` or `SHORT_TERM` for clarity and consistency.